### ISSUE TITLE
Loading error

### ISSUE THREAD

**Issue by haifengdashuaibi** [Tencent-Hunyuan/HunyuanVideo-Avatar#31]
Why is the progress bar stuck at 0 after the model loads successfully when I am running it? The computer's graphics card is 4090 and the running memory is 47.5G. In theory, the generation speed is not slow. I am running commands on a single GPU, and I have also reduced the image quality. The steps and frames are still stuck at 0 according to the test. The GPU is running full but still stuck at 0. I don't understand why something went wrong                 

guidance_scale:
t oee peyhue inue ceig te ois o ad il uy im corihnveeen eis ixpanding inputs for image tokens in lLaVa should bepatch size’ and " vision feature select strategye done in processing. PleaseI ce eent at tr in cine  wneee it iil alvrom y4. 46, the iogits1/50[3:31:03<172:21:27, 12663.02s/it]
sp\sample_gpu poor.py",line 108, in<module>
o ret n ee :Tencent workpiie yh:\Tencent_Fiie D:\reaudio.py", line 185, in predict= self.pipeline(promyF1e Sepintextlib.py",line 116, in decorate_contextwider amasa y, ine 1208 in .ask=prompt mask_input, text_states_2=prompt_enbeds_2 input, freqs_cos=freqs_cis[0],iettртопр1ine 1553, in  wapped_cal1_imp1eie on toand eai &are1ine 1562,in _ca11 imp1gllibisiaudio.py", line 587, in forward= self. txt_in(txmdule.py",line 1553,in _wapped_cal1_imp1i1e "D:\Tenceny11iblsite packatorch\nn\modules\module.py",1ine 1562,in ca11_implrefiner.py", 1ine 197, in forwarddules(module.py",line 1553,in _wrapped_call_imp1module.py",1ine 1562,in _cal1_implvllib\sittm to. a oe dr)in timestep embedding

**Comment by doogyhatts** [comment]
It went OOM. 24gb vram is insufficient, so your system hanged is what I guessed happened.

**Comment by haifengdashuaibi** [comment]
> 它 OOM 了？24GB 显存不够用，所以我猜你的系统挂了。

The running memory is 47.5G, but the GPU usage is not high. It took me 10 hours to generate it when I used very few frames and steps, but it took me over 100 hours when I used normal steps and frames. I don't understand why

INFO     | hymm_sp.sample_inference_audio:predict:180 -
                    prompt: Authentic, Realistic, Natural, High-quality, Lens-Fixed, A person sits cross-legged by a campfire in a forested area.
                image_path: assets/image/1.png
                audio_path: assets/audio/2.WAV
           negative_prompt: Aerial view, aerial view, overexposed, low quality, deformation, a poor composition, bad hands, bad teeth, bad eyes, bad limbs, distortion, blurring, Lens changes
                      seed: 128
                       fps: 25.0
               infer_steps: 25
             target_height: 704
              target_width: 768
             target_length: 129
            guidance_scale: 5.0

Expanding inputs for image tokens in LLaVa should be done in processing. Please add `patch_size` and `vision_feature_select_strategy` to the model's processing config or set directly with `processor.patch_size = {{patch_size}}` and processor.vision_feature_select_strategy = {{vision_feature_select_strategy}}`. Using processors without these attributes in the config is deprecated and will throw an error in v4.47.
Starting from v4.46, the `logits` model output will have the same type as the model (except at train time, where it will always be FP32)
100%|██████████████████████████████████████████████████████████████████████████████| 25/25 [9:41:34<00:00, 1395.79s/it]
2025-06-02 01:24:45.446 | INFO     | hymm_sp.sample_inference_audio:predict:225 - Success, time: 35574.02649354935
{'-r': '25.0', '-pix_fmt': 'bgr24'}
{'-pix_fmt': 'yuv420p', '-r': '25.0', '-vcodec': 'libx264', '-crf': '17', '-sws_flags': 'full_chroma_int+bitexact+accurate_rnd'}
D:\Tencent_workspace\ffmpeg-7.1.1-full_build\bin/ffmpeg.exe -y -r 25.0 -pix_fmt bgr24 -f rawvideo -s 768x704 -i - -pix_fmt yuv420p -r 25.0 -vcodec libx264 -crf 17 -sws_flags full_chroma_int+bitexact+accurate_rnd D:\Tencent_workspace\Tencent\HunyuanVideo-Avatar\results-single\8.mp4
